android/help/basic/config/index.md

```diff
@@ -47,6 +47,9 @@ Click on this to [adjust the keyboard height](adjust-keyboard-height).
 ## Spacebar Caption
 Click on this to [change the displayed keyboard name](spacebar-caption) on the spacebar.
 
+## Vibrate When Typing
+When enabled, the Keyman keyboard will provide haptic feedback (vibrate) when the user presses a key and generates output. The default preference is off.
```

Comment:
Question: is there an Android system preference for this as well? Should Keyman follow the Android system preference? Can you outline the various possible modes?

Reply:
On my Android 11 device, I don't see a system preference.
But Gboard has this under "Preferences"
* Toggle for "Haptic feedback on keypress"
* Slider for "Vibration strength on keypress"
    * "System default" is 0ms
    * Max is 100 ms